include/exadg/incompressible_navier_stokes/postprocessor/line_plot_data.h

```diff
@@ -170,48 +193,6 @@ struct LineHomogeneousAveraging : Line<dim>
   unsigned int averaging_direction; // x = 0, y = 1, z = 2
 };
 
-struct StatisticsData
-{
-  StatisticsData()
-    : calculate(false),
-      sample_start_time(0.0),
-      sample_end_time(1.0),
-      sample_every_timesteps(1),
-      write_output_every_timesteps(100)
```

Comment:
Because of this variable, I think we should introduce `TimeControlStatistics`. `TimeControl` should only contain the minimal set of parameters.